src/OpenLoco/Map/Track/Track.h

```diff
@@ -1,20 +1,14 @@
 #include "../../Types.hpp"
 #include "../Map.hpp"
+#include <utility>
```

Comment:
Why was this header introduced?

Reply:
its needed for std::pair. Previously this was got through some other header.